lib/trackjs/index.js

```diff
@@ -0,0 +1,17 @@
+/*jshint globalstrict:true, trailing:false, unused:true, node:true */
+"use strict";
+
+module.exports = {
+  apiVersion: 1,
+  name: 'TrackJS',
+  parse: function(headers, body) {
+    if(body.data && body.data.errors && body.data.errors.length > 0) {
+      var ret = {
+        message: 'TrackJS | ' + body.data.errors[0].message,
+        icon: 'logo',
+        errorLevel: body.data.errors[0].count > 1 ? 'error' : 'normal'
```

Comment:
Unlikely, I could change the condition to be `body.data.errors[0].count === 1 ? 'normal' : 'error'` if that's any better